### ISSUE TITLE
Consider removing all the resx schema unnecessary comments auto added by Visual Studio

### ISSUE THREAD

**Issue by carlossanlop** [dotnet/runtime#86906]
We are inconsistent in keeping or removing the unnecessary comments that are automatically added by Visyual Studio at the beginning of resx files.

It's not clear if we would open an issue in another repo, but we can manually address this in dotnet/runtime if we want.

Here is the full conversation where the details were discussed: https://github.com/dotnet/runtime/pull/85491#discussion_r1205003142

**Comment by eerhardt** [comment]
Note that we shouldn't do this unless VS stops adding it to the .resx file. I hate "fighting" VS. It would be really annoying to have to manually delete this section every time I changed a .resx file in a PR.

**Comment by danmoseley** [comment]
Yes unfortunately this realistically needs to be fixed in VS first. And VS probably uses the .NET Framework ResourceWriter, which can't reasonably be changed now, so it would have to do some tricks like editing the file before saving it.

More power to anyone who has a go at driving that on the VS side. 🙂
